.github/workflows/ruby-build.yml

```
@@ -0,0 +1,43 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby-Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Perhaps this would allow us to see the Action run, now?

```suggestion
  push:
  pull_request:
```

---

You can see runs in my repo. For example (https://github.com/ThomasKoppensteiner/march_hare/actions/runs/1309219243).